### ISSUE TITLE
replica ordering

### ISSUE THREAD

**Issue by ivukotic** [ssl-hep/ServiceX#766]
Sometimes replica returned by Rucio are wrongly ordered. We need them resorted in transformer sidecar. 
The easiest way is to give a list of preferred endpoints as an environment variable. 
This would also help ServiceXLite.

**Comment by gordonwatts** [comment]
Isn't this an issue with rucio and a bug that needs to be fixed there?

**Comment by ivukotic** [comment]
Yes, it's a Rucio bug and I have tried to get it fixed for years. It could take years more.

**Comment by ponyisi** [comment]
Closing the geolocation sorting as working at least for now.
